internal/sidecar/service/volumereplication.go

```diff
@@ -63,14 +63,24 @@ func (rs *ReplicationServer) EnableVolumeReplication(
 		klog.Errorf("Failed to get secret %s in namespace %s: %v", req.GetSecretName(), req.GetSecretNamespace(), err)
 		return nil, status.Error(codes.Internal, err.Error())
 	}
+	repReq := &csiReplication.EnableVolumeReplicationRequest{
+		ReplicationId: req.ReplicationId,
```

Comment:
attributes of gRPC requests should not be accessed directly, use `req.GetReplicationId()` instead. And everywhere else, specially in new code.